ui-tests/src/test/java/org/openmrs/reference/page/AddNewAccountPage.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.reference.page;
+
+import java.util.List;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+
+public class AddNewAccountPage extends Page {
+
+    public static final String URL = "/adminui/systemadmin/accounts/account.page";
+    private static final By FAMILY_NAME = By.cssSelector("#adminui-familyName-field");
+    private static final By GIVEN_NAME = By.cssSelector("#adminui-givenName-field");
+    private static final By GENDER_FEMALE = By.cssSelector("#adminui-gender-1-field");
+    private static final By ADD_USER_ACCOUNT = By.id("adminui-addUserAccount");
+    private static final By USER_NAME = By.cssSelector("#adminui-username-field");
+    private static final By PRIVILEGE_LEVEL = By.cssSelector("#adminui-privilegeLevel-field");
+    private static final By PASSWORD = By.cssSelector("#adminui-password-field");
+    private static final By CONFIRM_PASSWORD = By.cssSelector("#adminui-confirmPassword-field");
+    private static final By ADD_PROVIDER_ACCOUNT = By.id("adminui-addProviderAccount");
+    private static final By IDENTIFIER = By.id("adminui-identifier-field");
+    private static final By PROVIDER_ROLE = By.id("adminui-providerRole-field");
+    private static final By SAVE_BUTTON = By.id("save-button");
+
+    public AddNewAccountPage(Page parent) {
+        super(parent);
+    }
+
+    @Override
+    public String getPageUrl() {
+        return URL;
+    }
+
```

Comment:
Did you intensionally left two lines here ,  I  would have left one left to look more readable, otherwise this LG